napi-inl.h

```diff
@@ -3207,7 +3218,7 @@ inline void AsyncWorker::OnExecute(napi_env env, void* this_pointer) {
 }
 
 inline void AsyncWorker::OnWorkComplete(
-    napi_env env, napi_status status, void* this_pointer) {
+    napi_env /*env*/, napi_status status, void* this_pointer) {
```

Comment:
Why is it necessary to comment out some of the parameter names like this?

Reply:
Parameter of a function definition, that is not used in function body generate warning with `-Wunused-parameter` enabled.

https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html

Since library is header-only, even function definitions will be compiled with library user (as opposed to library author) flags. If library user has this warning enabled one will get warnings coming from library code, and to fix/hide them one will have to disable warnings for every compilation unit that includes this library, or wrap include with `#pragma push-ignore-pop` dance. Or fork/vendor library and fix warnings.